src/clj/rems/css/styles.clj

```diff
@@ -35,6 +35,10 @@
       (or attr-value (recur (rest attr-names))))))
 
 (def content-width (u/px 1200))
+(def logo-height-menu (u/px 40))
+(def logo-height (u/px 150))
+(def menu-height 56)
+(def recalculated-menu-height (u/px (int (+ menu-height 12))))
```

Comment:
Why is the recalculation done? I don't get the purpose from the name so we should consider a better name.

Also there should be no need to cast an int to an int.

Reply:
"cast an int to an int": well, I think this is the way it was done elsewhere in the styles file and also this is the way it doesn`t throw the error for me 

"Why is the recalculation done? I don't get the purpose from the name so we should consider a better name.' - I agree, the name could be better

Reply:
It's probably this garden bug that needs a workaround:

https://github.com/CSCfi/rems/blob/master/src/clj/rems/css/styles.clj#L794

However I think it just got fixed so we could just upgrade garden.

Reply:
Yep, https://github.com/noprompt/garden/issues/120 is fixed, but no release yet. So let's go with the workaround for now.

Reply:
Yeah but these are all ints. There is no decimal place.

Reply:
Right, it seems to work without int... weird, but ok

Reply:
thank you @opqdonut